documentation/modules/adding-source-provider.adoc

```diff
@@ -26,37 +26,174 @@ You can add {a-rhv} source provider by using the {ocp} web console.
 
 * {manager} CA certificate, unless it was replaced by a third-party certificate, in which case, specify the {manager} Apache CA certificate
 endif::[]
+ifdef::ova[]
+= Adding an Open Virtual Appliance (OVA) source provider
+
+You can add Open Virtual Appliance (OVA) files that were created by VMware vSphere as a source provider by using the {ocp} web console.
+
+include::snippet_ova_tech_preview.adoc[]
+endif::[]
+ifdef::ostack[]
+= Adding an {osp} source provider
+
+You can add an {osp} source provider by using the {ocp} web console.
+
+[NOTE]
+====
+Migration using {osp} source providers only supports VMs that use only Cinder volumes.
+====
+endif::[]
+ifdef::cnv[]
+= Adding a Red Hat {virt} source provider
```

Comment:
`Red Hat {virt}` looks like oVirt/RHV , we are talking about Openshift Virtualization (OCP/Kubevirt) right ?

@ahadas @RichardHoch ^^

Reply:
yes, {virt} refers to OpenShift Virtualization

Reply:
@yaacov Good question.  The attributes are rendered in the previews, so you can always check there to how they look in the final version -- in fact, it's a good idea to check there, too, since that will keep me honest.